src/components/AlgoliaLogo.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+export const AlgoliaLogo = () => {
```

Comment:
Most of these new `.tsx` are basically just a renaming of the `.js` version, and modified slightly to use JSX.